### DISCUSSION TITLE
Switch Plex Users

### DISCUSSION THREAD

**Discussion by jeremehancock** [croneter/PlexKodiConnect#1727] [Q&A]
I know that we can setup Profiles for each Plex user to allow switching users. However, I am curious is there a way to add a link in my theme to run the script that switches the logged in user? The only way that I see to do it is directly in the PlexKodiConnect settings. However, if this was accessible directly then it would be easy to add a menu item in my theme to allow switching between users easily. Thoughts?

**Comment by croneter** [comment]
You could somehow "trigger" the PKC following "entry" (NOT in the PKC settings): Add-ons -> Video Add-ons -> PlexKodiConnect -> "Log out user XYZ". I'm sure there are skins that let you do that. Or maybe you could even map a key to hit that menu entry.

**Reply by jeremehancock** [reply]
Thanks for the response. I just figured that out as well. :) It seems to work pretty well for what I want. The only two issues I have is:

1. The media takes a bit of time to switch. This is mostly an issue on my slower Android boxes.
2. Not easy to tell which user is currently logged in from my skin header

I think I am going to put together a simple addon that will run that script and maybe keep a loader going for a bit to give some time for things to update. Kinda hacky, but I think for my case it will work well. I might also do a mod to my theme to show the current logged in user if that is easy to access.

FYI: PlexKodiConnect is really freaking awesome! I really appreciate your work on it.
